docker-compose-production.yml

```diff
@@ -0,0 +1,40 @@
+app:
+  build: .
+web:
+  extends:
+    service: app
+  command: gunicorn -b 0.0.0.0:8000 --reload measurements.wsgi --workers 4 --timeout 120
+  ports:
+    - "8000"
+  links:
+    - redis
+    - rabbitmq
+https-portal:
+  image: steveltn/https-portal:1.0.0
+  ports:
+    - "80:80"
+    - "443:443"
+  links:
+    - web
```

Comment:
So apart from this there is the removal of the db container, correct?

I would see if there is a way to do that in the inherited config file if not then you should move the db container to config files of development, testing, staging and not add it to the production one.